### PR TITLE
Use Dynamic Leaderboard Layout

### LINKED ISSUE #195: Leaderboard Page: Fix Rank Overlap on Mobile
image

### PR DESCRIPTION
**Pull request by steets250**
Positions rank below name for thinner screens and is ready to contain badges.

Resolves #195

### CODE REVIEW

File: src/components/LeaderListItem/index.jsx [hunk 1]
```diff
@@ -14,12 +14,18 @@ const LeaderListItem = props => {
       <span className="placement">{props.placement}</span>
       <Avatar size={40} src={props.image} />
 
-      <span className="name">
-        <Link to={'/profile/' + props.uuid}>{props.name}</Link>
+      <span className="column left">
+        <div className="name">
+          <Link to={'/profile/' + props.uuid}>{props.name}</Link>
```
Comment (leestanley) [inline src/components/LeaderListItem/index.jsx:19]: can we add proptypes for the uuid it never got implemented. should just be a string

File: src/components/LeaderListItem/index.jsx [hunk 2]
```diff
@@ -14,12 +14,18 @@ const LeaderListItem = props => {
...
+        </div>
+        <div className="rank mobile">{getRank(props.exp)}</div>
       </span>
 
-      <span className="rank">{getRank(props.exp)}</span>
-      <span className="exp">{props.exp} points</span>
+      <span className="rank desktop">{getRank(props.exp)}</span>
+
```
Comment (leestanley) [inline src/components/LeaderListItem/index.jsx:25]: huh does running lint:fix not remove the extra blank lines above and below span?

File: src/components/LeaderListItem/index.jsx (hunk 1 above)
Comment (steets250) [reply src/components/LeaderListItem/index.jsx:18]: Yup, just added it in.

File: src/components/LeaderListItem/index.jsx (hunk 2 above)
Comment (steets250) [reply src/components/LeaderListItem/index.jsx]: Nope. It was in the original source but I removed the extra lines.

**Comment by leestanley** [comment]
once youre done with updating changes be sure to hit resolve conversation and re-request a review

**Pull request merged**